### ISSUE TITLE
Allow dynamic line rating when using SCLOPF

### ISSUE THREAD

**Issue by ClaraBuettner** [openego/eTraGo#724]
The sclopf is currently not compatible to dynamic line rating. The constraints need to be adjusted to consider s_max_pu. 

Since our status quo model does not include DLR, everything works for this scenario. But it should be adjusted before we start calculations for future scenarios.

**Comment by ClaraBuettner** [comment]
I think it would be also fine if this feature is not part of the upcoming release but will be added in the next one.
